index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/x-icon" href="assets/favicon.ico">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./styles/main.css">
+    <title>FOSS United | Campus Logo Generator</title>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <div class="bg-light-white" style="min-height: 100vh;">
+        <div class="container py-5">
+
+            <br>
+            <h3 class="text-center"><span class="text-blue">FOSS United</span> | Campus Logo Generator</h3>
+            <p class="text-secondary text-center">
+                A logo generator for FOSS United campus chapters.
+            </p>
+            <br>
+
+            <form class="form-group row" id="downloadForm">
+                <div class="col-md-2"></div>
+                <div class="col-md-4">
+                    <input type="text" id="collegeName" class="form-control mb-2 mb-md-0"
+                        placeholder="Enter your college name here" required>
+                </div>
+                <div class="col-md-3">
+                    <button type="button" id="toggleColor" class="btn btn-block btn-dark shadow-sm">Dark Theme</button>
```

Comment:
<img width="478" alt="Screenshot 2024-01-20 at 1 05 16 PM" src="https://github.com/fossunited/logo-generator/assets/60656677/411354ab-992a-46ef-9493-4e716ba0fd24">

The dark theme logo is different in colors. https://github.com/fossunited/logo-generator/blob/main/assets/fossclub_logo_dark.svg. Check this link for the logo, you can take the dark theme horizontal logo version from the Figma file too.

Reply:
Do push the change before resolving too.